src/cosmospy/_transaction.py

```diff
@@ -2,13 +2,15 @@
 
 import base64
 import hashlib
-import json
-from typing import Any
 
 import ecdsa
 
 from cosmospy._wallet import DEFAULT_BECH32_HRP, privkey_to_address, privkey_to_pubkey
-from cosmospy.typing import SyncMode
+import cosmospy.interfaces.any_pb2 as Any
```

Comment:
```suggestion
import cosmospy._interfaces.any_pb2 as Any
```

Can we make the `interfaces` package private?